cmd/tk/export.go

```diff
@@ -1,137 +1,88 @@
 package main
 
 import (
-	"bytes"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"log"
-	"os"
 	"path/filepath"
-	"strings"
 
-	"text/template"
-
-	"github.com/Masterminds/sprig/v3"
 	"github.com/go-clix/cli"
+	"github.com/pkg/errors"
 
+	"github.com/grafana/tanka/pkg/jsonnet/jpath"
 	"github.com/grafana/tanka/pkg/tanka"
 )
 
-// BelRune is a string of the Ascii character BEL which made computers ring in ancient times
-// We use it as "magic" char for the subfolder creation as it is a non printable character and thereby will never be
-// in a valid filepath by accident. Only when we include it.
-const BelRune = string(rune(7))
-
 func exportCmd() *cli.Command {
 	args := workflowArgs
-	args.Validator = cli.ValidateExact(2)
+	args.Validator = cli.ValidateFunc(func(args []string) error {
+		if len(args) < 2 {
+			return fmt.Errorf("expects at least 2 args, received %v", len(args))
+		}
+		return nil
+	})
 
 	cmd := &cli.Command{
-		Use:   "export <environment> <outputDir>",
-		Short: "write each resources as a YAML file",
+		Use:   "export <outputDir> <path> [<path>...]",
+		Short: "export environments found in path(s)",
 		Args:  args,
 	}
 
-	format := cmd.Flags().String("format", "{{.apiVersion}}.{{.kind}}-{{.metadata.name}}", "https://tanka.dev/exporting#filenames")
+	format := cmd.Flags().String(
+		"format",
+		"{{env.spec.namespace}}/{{env.metadata.name}}/{{.apiVersion}}.{{.kind}}-{{.metadata.name}}",
```

Comment:
This change possibly breaks existing scripts using `tk export`. Is it possible to keep the old functionality intact by default?